WebPhapp/backend/dummy_data/prescriptions.json

```diff
@@ -4,99 +4,99 @@
       "prescriptionID": 1,
       "patientID": 1,
       "drugID": 12,
-      "fillDates": ["10-05-2018"],
-      "writtenDate": "09-22-2018",
+      "fillDates": [1538697600000],
+      "writtenDate": 1537574400000,
       "quantity": 1,
       "daysFor": 7,
       "refillsLeft": 1,
       "prescriberID": 1,
       "dispenserID": 1,
       "cancelled": false,
-      "cancelDate": ""
+      "cancelDate": -1
     },
     {
       "prescriptionID": 2,
       "patientID": 1,
       "drugID": 13,
       "fillDates": [],
-      "writtenDate": "011-04-2018",
+      "writtenDate": 1541289600000,
       "quantity": 1,
       "daysFor": 14,
       "refillsLeft": 1,
       "prescriberID": 1,
       "dispenserID": 1,
       "cancelled": false,
-      "cancelDate": ""
+      "cancelDate": -1
     },
     {
       "prescriptionID": 3,
       "patientID": 1,
       "drugID": 14,
-      "fillDates": ["10-20-2018", "11-07-2018"],
-      "writtenDate": "10-13-2018",
+      "fillDates": [1539993600000, 1541548800000],
+      "writtenDate": 1539388800000,
       "quantity": 1,
       "daysFor": 14,
       "refillsLeft": 0,
       "prescriberID": 1,
       "dispenserID": 1,
       "cancelled": false,
-      "cancelDate": ""
+      "cancelDate": -1
     },
     {
       "prescriptionID": 4,
       "patientID": 1,
       "drugID": 15,
-      "fillDates": ["01-4-2019"],
-      "writtenDate": "01-2-2019",
+      "fillDates": [1546560000000],
+      "writtenDate": 1546416000000,
       "quantity": 1,
```

Comment:
Also, I thought quantity was a string. Might have missed this in a previous review.

Reply:
@mdulin2 I'll switch this to a string. A string actually makes more sense because it allows the user to specify a unit (mg, g, ml).

Reply:
Yes; that is what he had said initially.